crates/bevy_pbr/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 // FIXME(3492): remove once docs are ready
 #![allow(missing_docs)]
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
-#![forbid(unsafe_code)]
+#![allow(unsafe_code)]
```

Comment:
could you remove this from the crate level and allow it at the usage location?

Reply:
I tried leaving the forbid in place and putting an allow on the specific file, but it didn't seem to like that. What's the correct way to do this?

Reply:
You'd have to switch it to `#[deny]`; then `#[allow]` would work. In general I don't prefer using `#[forbid]` as it hurts debuggability for questionable gain--tools can easily find unsafe code without or without it.

Reply:
Fixed, thanks.